index.html

```diff
@@ -51,29 +51,22 @@ <h2>Archive:</h2>
         twttr.widgets.createTweet(archiveTweetId, tweetDiv, {});
       });
 
+      // Use latest tweet id to get the timestamp
+      var date = new Date(Math.floor(String(Math.floor(latestTweetId / 4194304))) + 1288834974657);
```

Comment:
why the double `Math.floor` + casting to string?

Reply:
Yes you're right, this should be enough. I am doing github pull requests for the first time, how do I change the line here? Should I close this PR and submit a new one? Thanks for the review.

Reply:
Figured it out :)